WcaOnRails/app/models/competition.rb

```diff
@@ -1161,6 +1159,14 @@ def confirmed=(new_confirmed_str)
     self.confirmed_at = new_confirmed ? (self.confirmed_at || Time.now) : nil
   end
 
+  def is_using_wca_live_for_scoretaking
+    self.is_using_wca_live_for_scoretaking?
+  end
+
+  def get_wca_live_link
```

Comment:
No `get_` prefix please, to keep it consistent with other methods.
For example, above we have `display_name` instead of `get_display_name` and `nearby_competitions` instead of `get_nearby_competitions`.